src/components/common/Directors.vue

```diff
@@ -591,75 +570,56 @@ export default class Directors extends Mixins(CommonMixin, DateMixin, DirectorMi
     mailAddressNew: ComponentIF,
     // form and components to edit an existing director:
     // (there is only 1 at a time but it's still an array)
-    editDirectorForm: Array<FormIF>,
-    baseAddressEdit: Array<ComponentIF>,
-    mailAddressEdit: Array<ComponentIF>
+    editDirectorForm: FormIF[],
+    baseAddressEdit: ComponentIF[],
+    mailAddressEdit: ComponentIF[]
   }
 
-  /** Effective date for querying current directors and appointing/ceasing them. */
-  @Prop({ default: '' })
```

Comment:
I moved these down in this file for consistency between vue files.

This is not a rule but I like to see files laid out this way:
- external getters (from store)
- props
- local variables
- internal getters
- created/mounted
- other main methods
- helper methods
- watchers
- emitters